docs/src/pages/components/selects/selects-pt.md

```diff
@@ -106,7 +106,7 @@ Exiba categorias com o componente `ListSubheader` ou com o elemento nativo `<opt
 
 ## Accessibility
 
-Para rotular corretamente seu campo  `Select` você precisa de um elemento extra com um `id` que contenha o rótulo desejado. Esse `id` precisa coincidir com o `labelId` do `Select`, por exemplo.
+Para rotular corretamente seu campo `Select` você precisa de um elemento extra com um `id` que contenha o rótulo desejado. Esse `id` precisa coincidir com o `labelId` do `Select`, por exemplo.
```

Comment:
I think this was just the result of running `yarn prettier`, it's removing an extra space. I'll see if I can edit this on the link you provided after the merge to avoid this happening again.